### PR TITLE
feat(resultsBasicTablePagination): AntD Components and Pagination

### PR DESCRIPTION
**Pull request by jarvisraymond-uchicago**
Jira Ticket: [VADC-466](https://ctds-planx.atlassian.net/browse/VADC-466)

### New Features
This updates the ResultsApp table to use pagination and column sorting by converting the table to use AntD components. This also brings the table much closer to the design, however final CSS styling work for release 1 will be done in ticket [VADC-467](https://ctds-planx.atlassian.net/browse/VADC-467)

This also includes adding the icons associated with each row's Job Status and an Actions dropdown with disabled options.

Additionally the tests and stories associated with the table have been updated as needed. 

### Improvements
Window.matchmedia object override has been globalized for the code base now. 

### Design
image

### Implementation
output

output

[VADC-466]: https://ctds-planx.atlassian.net/browse/VADC-466?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[VADC-467]: https://ctds-planx.atlassian.net/browse/VADC-467?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Analysis/GWASResultsV2/TestData/MatchMediaFix.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// See here for why this is needed:
+// https://stackoverflow.com/questions/39830580/jest-test-fails-typeerror-window-matchmedia-is-not-a-function
+
+Object.defineProperty(window, 'matchMedia', {
```
Comment (pieterlukasse) [inline src/Analysis/GWASResultsV2/TestData/MatchMediaFix.js:4]: do we need to keep the other workarounds you added before, like this one: https://github.com/uc-cdis/data-portal/blob/8ee6bbf6a78b2634ff3db4f5923b79b0c9ce231d/src/Analysis/GWASV2/Components/Diagrams/PhenotypeHistogram/Histogram.test.jsx#L7-L17 ?

Comment (pieterlukasse) [reply src/Analysis/GWASResultsV2/TestData/MatchMediaFix.js:4]: Or maybe you can reuse that one?

Comment (pieterlukasse) [reply src/Analysis/GWASResultsV2/TestData/MatchMediaFix.js:4]: Interesting: I actually removed the workaround above for Histogram.test.jsx and it still passes. I'm confused now...

**Review by pieterlukasse — COMMENTED** [review]
Looks good to me. A few minor comments on the matchMedia workaround. Also one regarding actions dropdown.

File: src/Analysis/GWASResultsV2/Views/Home/HomeTable/ActionsDropdown/ActionsDropdown.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Space, Dropdown, Button } from 'antd';
+import { EllipsisOutlined } from '@ant-design/icons';
+
+const items = [
+  {
+    key: '1',
+    label: (
+      <a target='_blank' rel='noopener noreferrer' href='#'>
+        Download
```
[the remark this answers is not in the document]
Comment (jarvisraymond-uchicago) [reply src/Analysis/GWASResultsV2/Views/Home/HomeTable/ActionsDropdown/ActionsDropdown.jsx:10]: The other options specified in the design, the tests associated with them, have been removed.

File: src/Analysis/GWASResultsV2/TestData/MatchMediaFix.js (hunk 1 above)
Comment (jarvisraymond-uchicago) [reply src/Analysis/GWASResultsV2/TestData/MatchMediaFix.js]: Per our discussion, this was moved to setupJest.js with global scope.

**Pull request merged**